app/serializers/search/chat_channel_membership_serializer.rb

```diff
@@ -2,7 +2,8 @@ module Search
   class ChatChannelMembershipSerializer < ApplicationSerializer
     attributes :id, :status, :viewable_by, :chat_channel_id, :last_opened_at,
                :channel_text, :channel_last_message_at, :channel_status,
-               :channel_status, :channel_type, :channel_username, :channel_name,
-               :channel_image, :channel_modified_slug, :channel_discoverable, :channel_messages_count
+               :channel_type, :channel_username, :channel_name, :channel_image,
+               :channel_modified_slug, :channel_discoverable,
+               :channel_messages_count
```

Comment:
`:channel_status` was originally duplicated and I cleaned up the spacing/formatting a little bit.